### ISSUE TITLE
Availability Grid Accessibility

### ISSUE THREAD

**Issue by jpdevries** [Thinkful/ui#324]
The current `<div>` based availability grid component is not accessible to keyboards or screen readers. It could also benefit from shift + click for range selection.

I'm working on an HTML&ndash;first AvailabilityGrid component that addresses these issues. Once it is a little more refined I'll start working on a "React driver" with the hopes of merging it upstream to here.
https://github.com/jpdevries/availability-grid/issues/1

![](http://j4p.us/093I2y0R322A/hero.gif)

The README thoroughly documents the usage and features and contains links to some screen casts
https://github.com/jpdevries/availability-grid
